weather_forecast/3dcorrection/unets/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM nvidia/cuda:11.3.1-cudnn8-devel-ubi8
+
+LABEL maintainer="Atos AI4Sim"
+
+RUN echo "alias ll='ls -la'" >> /root/.bashrc
+
+ENV https_proxy=http://129.183.4.13:8080
+ENV http_proxy=http://129.183.4.13:8080
+
+RUN dnf update -y \
+    && dnf install -y python39 \
+                      git \
+                      wget \
+                      sudo \
+    && dnf clean all \
+    && rm -rf /var/cache/dnf/*
+
+RUN ln -snf python3 /usr/bin/python
+RUN ln -snf pip3 /usr/bin/pip
+
+# RUN apt-get update -y \
+# && apt-get install -y sudo zsh
+
+ENV USER mluser
+ENV HOME /home/${USER}
+RUN adduser --create-home ${USER} \
+    && echo "${USER} ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
+# RUN adduser --disabled-password --gecos "" ${USER} \
+    # && echo "${USER} ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
+USER ${USER}
+WORKDIR ${HOME}
+ENV PATH="${HOME}/.local/bin:${PATH}"
+RUN echo "export PATH=${PATH}:${HOME}/.local/bin" >> ${HOME}/.bashrc
+
+RUN pip install --upgrade pip
```

Comment:
Why not using the same way to install dependencies ? For the other UC, the nox target is used, it avoids to duplicate in the dockerfile the list of the dependencies already listed in the `requirement.txt` file.